src/rbac/domain/project/entity/project_role.py

```
@@ -0,0 +1,41 @@
+from enum import Enum, auto
+from typing import Set
+from __future__ import annotations
```

Ruff (0.8.2)

`from __future__` imports must occur at the beginning of the file

(F404)